src/sidebar/components/sidebar-content.js

```diff
@@ -18,7 +19,7 @@ import ThreadList from './thread-list';
  * @typedef SidebarContentProps
  * @prop {() => any} onLogin
  * @prop {() => any} onSignUp
- * @prop {Object} [frameSync] - Injected service
+ * @prop {Object} [bridge] - Injected service
```

Comment:
Since `Bridge` is already an ES class, it would be easy to make the type more specific at this point too:

```
@prop {import('../../shared/bridge').default} bridge
```